frontend/src/app/menu/menu.component.html

```diff
@@ -1,50 +1,113 @@
-<nav class="navbar" *ngIf="currentUser" aria-label="primary navigation">
+<nav class="navbar is-primary" aria-label="primary navigation">
     <div class="container is-fluid">
+        <!-- skip link for keyboard navigation.
+            Complicated setup because the menu is outside of router-outlet and so
+            it can't use [routerLink]="." -->
+        <a class="skip-to-content-link" *ngIf="route$ | async as route"
+            [routerLink]="route.url" [queryParams]="route.queryParams" fragment="main">
+            Skip to content
+        </a>
         <div class="navbar-brand">
             <a class="navbar-item" [routerLink]="['/home']">
                 <img src="/assets/logo.png" aria-hidden="true" width="50" height="50">
                 I-ANALYZER
             </a>
 
-            <a (click)="toggleMenu($event)" role="button" [ngClass]="{'navbar-burger': true, 'is-active': menuOpen}" aria-label="menu" aria-expanded="false">
-                    <span aria-hidden="true"></span>
-                    <span aria-hidden="true"></span>
-                    <span aria-hidden="true"></span>
+            <a  role="button" tabindex="0"
+                (click)="toggleMenu($event)" (keydown.enter)="toggleMenu($event)"
+                class="navbar-burger" [class.is-active]="menuOpen"
+                aria-label="menu" [attr.aria-expanded]="menuOpen"
+                data-target="navigation-menu">
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
+                <span aria-hidden="true"></span>
```

Comment:
Why the repeated span?

Reply:
I thought this was a mistake, but this is how the bulma documentation does it. My best guess is that this is how they create the "hamburger icon". I haven't tried removing it though.